Add records errors section

remilejeune opened the pull request.

remilejeune committed:
Add records errors section

Aquassaut commented on source/includes/_dataset_status.md:
````diff
@@ -56,21 +56,59 @@ The dataset status object that applies to the given dataset. See [Dataset status
 
 ```json
 {
     "published": True,
````
true, not True

Aquassaut commented on source/includes/_dataset_status.md:
````diff
@@ -38,7 +38,7 @@ curl https://yourdomain.opendatasoft.com/api/management/v2/datasets/da_XXXXXX/st
 ```json
 {
-    "published": True,
+    "published": False,
````
false, not False

Aquassaut commented on source/includes/_dataset_status.md:
````diff
@@ -56,21 +56,59 @@ The dataset status object that applies to the given dataset. See [Dataset status
+
+```json
+{
+    "published": True,
````
true not True

Aquassaut commented on source/includes/records_errors.md:
```diff
@@ -0,0 +1,75 @@
+
+### Returns
+
+The list of records errors object that applies to the given dataset. See [records errrors object](#records_errors) for specific attributes and the list of objects.
```
The list of record error objects that apply to the given dataset
In square brackets, there's a r too many to "errrors".

meganelacheny commented on source/includes/_dataset_status.md:
```diff
@@ -56,21 +56,59 @@ The dataset status object that applies to the given dataset. See [Dataset status
-No specific attribute
+Attribute | Description
+--------- | -----------
+records_errors <br> *[records errrors object](#records_errors)* <br> <em class="expandable">expandable</em> | Errors which occurred during the processing. Can be from a processor, or a type conversion. This attribute is present only if the dataset is published.
```
"[records **errors** object]"

meganelacheny commented on source/includes/_datasets.md:
```diff
@@ -35,9 +35,8 @@ Attribute | Description
--`dataset_processing_status` <br> *[dataset processing status object](#dataset-processing-status)* <br> <em class="expandable">expandable</em> | Keyword indicating if the dataset is waiting to be published, currently being published or if it encountered errors during last publishing
+`metas` <br> *[metadata object](#dataset-metadata)* | Dictionary of attributes about the dataset like a title, a description, keywords, that make it easily searchable through the portal's catalog
+`last_modified` <br> *datetime*  | Date when the dataset's configuration was last edited
+`default_security` <br> *object* |
```
Why is there no description for default security?

remilejeune replied:
This property should not be there ...

meganelacheny commented on source/includes/records_errors.md:
```diff
@@ -0,0 +1,75 @@
+
+Attribute | Description
+--------- | -----------
+`record_id` <br> *string* | Id of the record after processing
```
Could you please replace "Id" by "Identifier"?

meganelacheny commented on source/includes/records_errors.md:
```diff
@@ -0,0 +1,75 @@
+
+The record error describes errors which occured during the processing. An error comes from:
+
+* a processor: when a value is valid or an operation failed
```
"when a value is **in**valid"

meganelacheny commented on source/includes/records_errors.md:
```diff
@@ -0,0 +1,75 @@
+
+## The record error object
+
+The record error describes errors which occured during the processing. An error comes from:
```
"which **occurred** during the processing"

meganelacheny commented on source/includes/records_errors.md:
```diff
@@ -0,0 +1,75 @@
+Attribute | Description
+--------- | -----------
+`record_id` <br> *string* | Id of the record after processing
+`processor_uid` <br> *string* | Uid of the processor who generates the error
```
"**Unique identifier** of the processor **which** generates the error"

meganelacheny commented on source/includes/records_errors.md:
```diff
@@ -0,0 +1,75 @@
+--------- | -----------
+`record_id` <br> *string* | Id of the record after processing
+`processor_uid` <br> *string* | Uid of the processor who generates the error
+`field_uid` <br> *string* | Uid of the field involved in the error
```
Could you please replace "Uid" by "Unique identifier"?

remilejeune committed:
Fix PR requested changes

remilejeune committed:
Remove save_version status and action

remilejeune commented:
I fixed the comments and deleted the save version part, it does not make sense to have a "save version" action but not api to list the versions and download them.
It will be dedicated development.

remilejeune committed:
Replace "status" with "name". Abort action does not return a job_id

remilejeune committed:
Fix typo
